### ISSUE TITLE
Puppeteer FileChooser.accept is not able to find files uploaded to browserless

### ISSUE THREAD

**Issue by Zhang** [browserless/chrome#877]
**Describe the bug**
Browserless is unable to find my uploaded file when using puppeteer's fileChooser.accept method

**To Reproduce**
```
const fs = require('fs');
const puppeteer = require('puppeteer');
const request = require('request');

const browser = await puppeteer.connect({ browserWSEndpoint: `ws://localhost:3001`, slowMo: 15 });
const page = await browser.newPage()

await page.goto('https://cgi-lib.berkeley.edu/ex/fup.html');
const file = fs.readFileSync(YOUR PATH + '/form.pdf');

request.post({
  url: `http://localhost:3001/workspace`,
  formData: { custom_file: { value: file, options: { filename: 'form.pdf', contentType: 'application/pdf' } } },
}, async (err, res) => {
  const [fileChooser] = await Promise.all([
    page.waitForFileChooser(),
    page.click('input[type="file"]'),
  ]);
  
  await fileChooser.accept([JSON.parse(res.body)[0].path]);

  await page.click('input[type="submit"]');
});
```
**Expected behavior**
Expected behavior is that the file is uploaded properly. This works when running puppeteer headless w/out browserless. Instead, I receive the error: ```ENOENT: no such file or directory, open '/usr/src/app/workspace/form.pdf``` 

**Desktop (please complete the following information):**
 - Device: Mac
 - OS: 10.12.6

**Additional context**
Attempted the exact script attached on puppeteer 5.0.0 and browserless/chrome:1.36.0-puppeteer-5.0.0 run with the command ```docker run --rm -p 3001:3000 -e  "CONNECTION_TIMEOUT=600000 MAX_CONCURRENT_SESSIONS=10 USE_CHROMLE=true" browserless/chrome:1.36.0-puppeteer-5.0.0```

I've gotten the same error prior when attempting w/ puppeteer 2.1.1 as well

**Comment by joelgriffith** [comment]
TL;DR: All puppeteer versions seem to validate that the files being uploaded are on the file-system that puppeteer resides on (not on the remote file-system for browserless' case). You can run whatever version of browserless you want, so long as your puppeteer version is 3.0.0 or less -- and you don't need to upload the files to the docker container of browserless prior!

Long:
Ok, so after some deep diving, this is actually a bug in recent versions of puppeter. Puppeteer, less than 3.0.1, does a really interesting trick where they [read the files from the puppeteer-side and load them via `data` URLs into chrome](https://github.com/puppeteer/puppeteer/blob/v3.0.0/src/JSHandle.js#L335). That means it works both on `connect` as well as `launch`, and you _don't_ need to upload the files to the browserless container. You can run whatever version of the service you want (I tried on several) so long as _your_ puppeteer version is less than 3.0.1.

[Starting in 3.0.1, puppeteer no longer reads these files into chromium via data-URI's](https://github.com/puppeteer/puppeteer/blob/v3.0.1/src/JSHandle.ts#L301). Instead, it opts to let chromium read them from the file-system, which breaks `connect`ed browser sessions since puppeteer validates that the files are on _it's_ file-system, but not on the file-system that's running chromium.

**Comment by joelgriffith** [comment]
You can also read this comment which has another potential workaround: https://github.com/puppeteer/puppeteer/issues/4405#issuecomment-618581699

**Comment by Zhang** [comment]
I wanted to follow up on this and note that 2.1.1 is likely the best version of puppeteer to use w/ fileChooser in this way.

2.0.0 has the multiple connections bug because it ships w/ an incompatible version of chromium
